discord/ext/commands/errors.py

```diff
@@ -118,6 +118,12 @@ class CheckFailure(CommandError):
     """Exception raised when the predicates in :attr:`.Command.checks` have failed."""
     pass
 
+class PrivateMessageOnly(CheckFailure):
```

Comment:
You're going to need to add this to __all__

Reply:
Yes, my bad.